modules/systemSettings.js

```diff
@@ -0,0 +1,321 @@
+function SystemSettings() {
+
+  this.systemsettings = {
+    'accessibility.invert': false,
+    'accessibility.screenreader': false,
+    'alarm.enabled': false,
+    'app.update.interval': 86400,
+    'audio.volume.alarm': 15,
+    'audio.volume.bt_sco': 15,
+    'audio.volume.dtmf': 15,
+    'audio.volume.content': 15,
+    'audio.volume.notification': 15,
+    'audio.volume.tts': 15,
+    'audio.volume.telephony': 5,
+    'bluetooth.enabled': false,
+    'camera.shutter.enabled': true,
+    'geolocation.enabled': true,
+    'keyboard.layouts.english': true,
+    'keyboard.layouts.dvorak': false,
+    'keyboard.layouts.otherlatins': false,
+    'keyboard.layouts.cyrillic': false,
+    'keyboard.layouts.arabic': false,
+    'keyboard.layouts.hebrew': false,
+    'keyboard.layouts.zhuyin': false,
+    'keyboard.layouts.pinyin': false,
+    'keyboard.layouts.greek': false,
+    'keyboard.layouts.japanese': false,
+    'keyboard.layouts.polish': false,
+    'keyboard.layouts.portuguese': false,
+    'keyboard.layouts.spanish': false,
+    'keyboard.vibration': false,
+    'keyboard.clicksound': false,
+    'keyboard.autocorrect': true,
+    'keyboard.wordsuggestion': true,
+    'keyboard.current': 'en',
+    'language.current': 'en-US',
+    'lockscreen.passcode-lock.code': '0000',
+    'lockscreen.passcode-lock.timeout': 0,
+    'lockscreen.passcode-lock.enabled': false,
+    'lockscreen.notifications-preview.enabled': true,
+    'lockscreen.enabled': true,
+    'lockscreen.locked': true,
+    'lockscreen.unlock-sound.enabled': false,
+    'mail.sent-sound.enabled': true,
+    'message.sent-sound.enabled': true,
+    'phone.ring.keypad': true,
+    'ring.enabled': true,
+    'screen.automatic-brightness': true,
+    'screen.brightness': 1,
+    'screen.timeout': 60,
+    'vibration.enabled': true,
+    'wifi.enabled': true,
+    'wifi.screen_off_timeout': 600000,
+    'wifi.disabled_by_wakelock': false,
+    'wifi.notification': false,
+    'wifi.connect_via_settings': false,
+    'icc.displayTextTimeout': 40000,
+    'icc.inputTextTimeout': 40000
+  };
+
+
+}
+
+
+/**
+ *@description SystemSettings Constructor
+ *
+ */
+SystemSettings.prototype.initialize = function() {
+
+  try {
+    this.loadFromDevice();
```

Comment:
Make naming consistent across modules that use this pattern (settings, systemSettings, and history)